src/05-objects.js

```diff
@@ -35,7 +39,11 @@ function createObjectWithValue(val) {}
  *  createObjectWithKey("left");
  *  //> { left: false }
  */
-function createObjectWithKey(key) {}
+function createObjectWithKey(key) {
+  let Object = {};
```

Comment:
be very careful here` Object ` with a capital O initializes a new object and is not the same as ` let object = {} `